jaxrs-api/src/main/java/jakarta/ws/rs/container/CompletionCallback.java

```diff
@@ -21,7 +21,7 @@
  * <p>
  * A completion callback is invoked when the whole request processing is over, i.e. once a response for the request has
  * been processed and sent back to the client or in when an unmapped exception or error is being propagated to the
- * container.
+ * default exception mapper.
  * </p>
```

Comment:
This can be made more accurate. It will be propagated to _the default exception mapper_ only if it hasn't been overridden. So something like "propagated to an exception mapper, possibly the default one if not overridden".

Reply:
I am fine with updating the javadoc. However, I do not understand the comment. It cannot be propagated to any other exception mapper but the default one. Jakarta REST does not provide a mechanism to override the default exception mapper, per se, it merely allows for registering a custom exception mapper. But if the custom exception mapper applies, there is no unmapped exception at this point, the mapped throwable is not used for the `CompletionCallback` (in 3.0).

Perhaps we can replace "unmapped exception" by "exception unmapped by a registered exception mapper"?